content/docs/start/visualization.md

```diff
@@ -0,0 +1,108 @@
+---
+title: 'Get Started: Visualization with Plots'
+---
+
+# Get Started: Visualization with Plots
+
+In this section, we'll add visualization to the [`example-dvc-experiments`][ede]
+project (explored [previously](/doc/start/experiments)). If you would like to
+try these yourself, please refer to the project. [README] about how to install.
+
+[ede]: https://github.com/iterative/example-dvc-experiments
+[readme]:
+  https://github.com/iterative/example-dvc-experiments/blob/main/README.md
+
+### Creating plots from tabular data
+
+A useful plot to show the classification performance is the [confusion matrix].
+In order to produce it, DVC expects a CSV **plots file** in the form:
+
+```csv
+actual,predicted
+0,0
+0,2
+...
+```
+
+> We added a [loop] comparing the results to generate this file from the
+> predictions.
+
+[loop]:
+  https://github.com/iterative/example-dvc-experiments/blob/main/src/train.py#L123
+[confusion matrix]: https://en.wikipedia.org/wiki/Confusion_matrix
+
+Running the experiment with `dvc exp run` will produce `plots/confusion.csv`.
+Use `dvc plots show` to present it as an HTML file, and open it in the browser:
+
+```dvc
+$ dvc plots show plots/confusion.csv --template confusion \
+                                     -x actual -y predicted
+file:///Users/.../example-dvc-experiments/plots/confusion.json.html
+```
+
+![confusion matrix](/img/start_visualization_confusion1.png)
+
+## Displaying user-generated plot images
+
+Let's produce another plot to see misclassified examples from each class. This
+procedure generates the misclassification examples from the validation data and
+arranges them into a _confusion table_ that shows the correct label, and
+misclassification sample. The code to generate an image from a set of training
+images is omitted here but you can find the code in [the example
+project.][misclassified-example-code]
+
+[misclassified-example-code]:
+  https://github.com/iterative/example-dvc-experiments/blob/48b1e5078c957f71674c00f416290eaa3b20b559/src/util.py#L49
+
+```dvc
+$ dvc plots show plots/confusion.png
+```
+
+![confusion image](/img/start_visualization_confusion2.png)
+
+## Autogenerating plots from deep learning code
+
+An important issue for deep learning projects is to observe in which epoch do
+training and validation loss differ. DVC helps in that regard with its Python
+integrations to deep learning libraries via [DVCLive].
+
+The example project uses Keras to train a classifier, and we have a DVCLive
+callback that visualizes the training and validation loss for each epoch. We
+first import the callback from DVCLive.
+
+```python
+from dvclive.keras import DvcLiveCallback
+```
+
+Then we add this callback to the `fit()` call.
```

Comment:
_From https://github.com/iterative/dvc.org/pull/3050#pullrequestreview-937490083_

> I don't think so. It's Keras specific, and Keras users already know to use those callbacks.

Yes but this page is not for Keras users. We just happen to use Keras in the example project.

```suggestion
Then we add this callback to the
[`fit` method](https://keras.io/api/models/model_training_apis/#fit-method)
call.
```